### PR TITLE
Survey-project by Idah Collin & Emmy Dieden

### PR DESCRIPTION
**Pull request by emmydieden**
Deployed project:

https://survey-project-collin-dieden.netlify.app/

### CODE REVIEW

**Review by smExlex — COMMENTED** [review]
First of all, what a nice looking survey! I really enjoyed the calm colours. 🤗

I had a look at the app in different screen sizes and overall it looks good. When resizing the text to 200% I did notice some loss of content of both header and footer, but it was still possible to fill in the form. I had no trouble tabbing through the survey, the buttons had a nice and clear focus colour.

Your code is clean and easy to read, great with comments to explain what everything does.

I could easily check off every requirement of this project - you did a great job. Well done! 👌

File: src/components/Survey/Activity/Activity.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  //Rendering a select drop-down
+  return (
+    <div className="activity-select">
+      <p id="activity-description">Which adventure are you longing for?</p>
+      <select 
+        className="select-style"
+        value={value} 
+        onChange={selectedActivity}
+        aria-labelledby='activity-description' //Added for accessibility purposes, as label was removed. (for styling reasons)
```
[the remark this answers is not in the document]
Comment (JuliaHolm) [reply src/components/Survey/Activity/Activity.jsx:14]: I agree with you Susanne!

File: README.md [hunk 2]
```diff
@@ -1,37 +1,15 @@
...
 ### The Problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+As a pair we were new to using state and are relatively new to React. Problems were solved by team-work, going through course-material and chatGPT.
 
-### View it live
+A sketch was made showing the structure of the components and basic design. We changed some of this to make our project more clean.
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+With more time we would have added the map-method to the radio buttons, validation (to require answers), e-mail input and we would also have hidden the header-information when the summary is shown.
```
Comment (JuliaHolm) [inline README.md:11]: I was going to recommend the last two, e-mail input and hide the header information in the summary :) Now that I saw it here I understand that you considered it but didn't have time to do it. Maybe you can do it afterwards to fullfil the page as you imagined it 🥰

File: src/App.jsx [hunk 3]
```diff
@@ -1,3 +1,19 @@
+//Overview of components structure:
+//App.jsx (parent)
+//Header.jsx, Survey.jsx, Footer.jsx (children of App.jsx)
+//Name.jsx, Activity.jsx, When.jsx (children of Survey.jsx)
```
Comment (JuliaHolm) [inline src/App.jsx:4]: Nice idea with a overview with comments! This is something I will start doing from now on.

File: src/App.jsx [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+import {Survey} from "./components/Survey/Survey"
+import {Header} from "./components/Header/Header"
+import {Footer} from "./components/Footer/Footer"
+
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
-};
+  return (
+  
+  <div className="body-container">
```
Comment (JuliaHolm) [inline src/App.jsx:13]: Good class names :)

File: src/components/Footer/footer.css [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.footer {
+    text-align: center;
+    color: white;
+}
```
[the remark this answers is not in the document]
Comment (JuliaHolm) [reply src/components/Footer/footer.css:4]: I also saw that the background don't fill the whole screen when scrolling down. I think you can fix that by adding position: fixed; and some other attributes. Same goes for header and footer. If you want to, I can help and check exactly how I did it in the end for it to work 😄 just tell me 👍

File: src/components/Survey/Activity/Activity.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+      >
+      <option disabled value="">Choose an adventure!</option>
```
Comment (JuliaHolm) [inline src/components/Survey/Activity/Activity.jsx:16]: disabled value was a good add on to the first option :)

**Comment by JuliaHolm** [comment]
I tried to point out the good choices that was made but also give some advice. Susanne already pointed out a lot of good thinks that I agree with. I really like the color with less opacity you added in the background behind the survey, I think that it makes the design whole. Be proud of what you achieved in this project!

**Review by InnaKokic — COMMENTED** [review]
I can see that Susanne and Julia already gave you some detailed comments and I don't have that much too add without it being redundant! You have created a really good looking site and you should be proud! 

You have a good structure of your components and it's easy to follow the flow of it all. Your code is also super clean and your variables have good names. Overall great job you guys!!

**Pull request closed without merging**